README.md

```diff
@@ -193,11 +193,13 @@ For Python unit tests, run:
 ```sh
 docker-compose run --rm py.test
 docker-compose run --rm flake8  # linting
+docker-compose run --rm ptw
 ```
 
 For JS unit tests, run:
 ```sh
 docker-compose run --rm npm test
+docker-compose run --rm npm run test:watch
```

Comment:
Looks good to me, maybe we should include a comment about what the difference between the two of these is? Or is it somewhat obvious from the way the command is structured?

Reply:
@jeremiak I think thats a good idea. Ill add that.